frontend/packages/ceph-storage-plugin/integration-tests/tests/1-install/installFlow.scenario.ts

```diff
@@ -68,40 +74,69 @@ describe('Testing OCS Subscription', () => {
   });
 });
 
-describe('Test creation of Storage Cluster', () => {
-  it(
-    'creates a storage cluster',
-    async () => {
-      const { selectedNodes } = await Installer.createStorageCluster();
-      browser.sleep(2 * SECOND);
-      const text = await crudView.resourceTitle.getText();
-      expect(text).toEqual(STORAGE_CLUSTER_NAME);
-      // Verify all the nodes have the required labels
-      // Wait for 5 seconds for label to apply
-      await browser.sleep(5 * SECOND);
-      let nodes: string[] = await Promise.all(selectedNodes);
-      // Data syntax Node\nN\n<node-name>
-      nodes = nodes.map((node) => node.split('\n')[2]);
-      nodes.forEach((node) => expect(verifyNodeLabels(node, OCS_NODE_LABEL)).toBeTruthy());
-      const storageCR = JSON.parse(
+if (TEST_PLATFORM === Platform.OCP || (TEST_PLATFORM === Platform.OCS && MODE === Mode.CONVERGED)) {
+  describe('Test creation of Converged Storage Cluster', () => {
+    it(
+      'creates a storage cluster',
+      async () => {
+        const { selectedNodes } = await Installer.createConvergedStorageCluster();
+        browser.sleep(2 * SECOND);
+        const text = await crudView.resourceTitle.getText();
+        expect(text).toEqual(STORAGE_CLUSTER_NAME);
+        // Verify all the nodes have the required labels
+        // Wait for 5 seconds for label to apply
+        await browser.sleep(5 * SECOND);
+        let nodes: string[] = await Promise.all(selectedNodes);
+        // Data syntax Node\nN\n<node-name>
+        nodes = nodes.map((node) => node.split('\n')[2]);
+        nodes.forEach((node) => expect(verifyNodeLabels(node, OCS_NODE_LABEL)).toBeTruthy());
+        const storageCR = JSON.parse(
+          execSync(
+            `kubectl get storageclusters ${STORAGE_CLUSTER_NAME} -n ${OCS_NS} -o json`,
+          ).toString(),
+        );
+        const scFromCR =
+          storageCR?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec?.storageClassName;
+        const size =
+          storageCR?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec?.resources?.requests
+            ?.storage;
+        const defaultSC = execSync(`kubectl get storageclasses | grep -Po '\\w+(?=.*default)'`)
+          .toString()
+          .trim();
+        expect(size).toEqual('512Gi');
+        expect(defaultSC).toEqual(scFromCR);
+      },
+      16 * MINUTE,
+    );
+  });
+}
+
+if (TEST_PLATFORM === Platform.OCP || (TEST_PLATFORM === Platform.OCS && MODE === Mode.EXTERNAL)) {
+  describe('Test creation of External Storage Cluster', () => {
+    beforeAll(async () => {
+      await Installer.createExternalStorageCluster();
+    });
```

Comment:
Why there is a difference in design of tests for converged and independent ?
I am assuming a test case for cluster creation as well.

Reply:
I don't understand your concern.

Reply:
Is it clarifying ?
```suggestion
    it(
      'creates a storage cluster',
      async () => {
```